### ISSUE TITLE
unprovable assertion not highlighted by Dafny after loop with no body

### ISSUE THREAD

**Issue by yannickmoy** [dafny-lang/dafny#604]
After the following loop whose body is not provided, Dafny does not highlight that the assertion x==25 is unprovable, although it can deduce that x==100 from the loop guard and invariant. Is that because it does not attempt to prove properties after an unimplemented loop? That seems at odds with the fact it generates a decreases and havoc for that loop.

I get the same in VSCode and on the command-line, Dafny only warns about the missing body for the loop:
loops.dfy(3,4): Warning: note, this loop has no body

```
method Inv() returns (x: int) {
    x := 0;
    while x < 100
        invariant 0 <= x <= 100
    assert x == 25;
    return x;
}
```

**Comment by davidcok** [comment]
The command-line does produce an assertion error, but VSCode still does not show it as a Problem or highlight it.

**Comment by davidcok** [comment]
The version of Dafny.exe shipped with current VSCode does not (when run standalone on the command-line) report the assertion violation following the missing while body.
I found that some combination of the following enabled the assertion to be highlighted in VSCode.
1) Setting a current build of Dafny as the Dafny root in the preferences.
2) Creating a workspace in which to hold the test file (because VSCode threatens that some features are not enabled otherwise).
3) Restarting VSCode a few times.
I think that probably #1 along with #3 to cause VSCode to use the new path is the key.
However, a new release of VSCode with a current build of Dafny should fix the problem out of the box. 

I also lodged Issue#60 with the VSCode-dafny project to request that the dafny version and output be visible.
